tests/unit/integrations/watchers/indexable-permalink-watcher-test.php

```diff
@@ -114,21 +114,44 @@ public function test_reset_permalinks() {
 		$this->instance->expects( 'get_post_types' )->once()->andReturn( [ 'post' ] );
 		$this->instance->expects( 'get_taxonomies_for_post_types' )->once()->with( [ 'post' ] )->andReturn( [ 'category' ] );
 
-		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'post', 'post' )->once();
-		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'post-type-archive', 'post' )->once();
-		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'term', 'category' )->once();
-		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'user' )->once();
-		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'date-archive' )->once();
-		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'system-page' )->once();
+		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'post', 'post', null )->once();
+		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'post-type-archive', 'post', null )->once();
+		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'term', 'category', null )->once();
+		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'user', null, null )->once();
+		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'date-archive', null, null )->once();
+		$this->indexable_helper->expects( 'reset_permalink_indexables' )->with( 'system-page', null, null )->once();
```

Comment:
Not sure these should pass `null` for the reason value. I'm assuming a reason should always be set. Looking at similar tests, for example the ones for the `homeurl`, I think the real value should be passed. Unless I'm missing something 🙂

Reply:
This has been fixed